.buildkite/pipelines/quality-gates/pipeline.tests-qa.yaml

```diff
@@ -3,18 +3,6 @@
 # A failure in this pipeline build will prevent further progression to the subsequent stage.
 
 steps:
-  - label: ":pipeline::kibana::seedling: Trigger SLO check"
-    trigger: "serverless-quality-gates" # https://buildkite.com/elastic/serverless-quality-gates
-    build:
-      message: "${BUILDKITE_MESSAGE} (triggered by pipeline.tests-qa.yaml)"
-      env:
-        TARGET_ENV: qa
-        CHECK_SLO: true
-        CHECK_SLO_TAG: kibana
-        CHECK_SLO_WAITING_PERIOD: 10m
-        CHECK_SLO_BURN_RATE_THRESHOLD: 0.1
```

Comment:
@delanni I can't remember why you added these two extra environment variables in QA and not in Staging. Do you recall? Right now I just copied them to Prod, but maybe since they were not in Staging, they are not needed in Prod either

Reply:
I think this is a more forgiving setup than the default values.
I think @ramonbutter knows more about the defaults/suggestions.

Reply:
I removed them from Prod for now